### PR TITLE
+faraday.0.3.0

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

**Comment by samoht** [comment]
Failure when building `faraday-async`:

```
#=== ERROR while installing faraday-async.0.3.0 ===============================#
# opam-version         1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os                   linux
# command              jbuilder build -p faraday-async -j 2
# path                 /home/opam/.opam/system/build/faraday-async.0.3.0
# compiler             system (4.02.3)
# exit-code            1
# env-file             /home/opam/.opam/system/build/faraday-async.0.3.0/faraday-async-36-3fc2d0.env
# stdout-file          /home/opam/.opam/system/build/faraday-async.0.3.0/faraday-async-36-3fc2d0.out
# stderr-file          /home/opam/.opam/system/build/faraday-async.0.3.0/faraday-async-36-3fc2d0.err
### stderr ###
#     ocamldep async/faraday_async.dependsi.ocamldep-output
#     ocamldep async/faraday_async.depends.ocamldep-output
#       ocamlc async/faraday_async.{cmi,cmti} (exit 2)
# (cd _build/default && /usr/bin/ocamlc.opt -w -40 -g -bin-annot -I /home/opam/.opam/system/lib/async -I /home/opam/.opam/system/lib/async_extra -I /home/opam/.opam/system/lib/async_kernel -I /home/opam/.opam/system/lib/async_rpc_kernel -I /home/opam/.opam/system/lib/async_unix -I /home/opam/.opam/system/lib/bin_prot -I /home/opam/.opam/system/lib/bytes -I /home/opam/.opam/system/lib/core -I /home/opam/.opam/system/lib/core_kernel -I /home/opam/.opam/system/lib/faraday -I /home/opam/.opam/system/lib/fieldslib -I /home/opam/.opam/system/lib/num -I /home/opam/.opam/system/lib/ocplib-endian -I /home/opam/.opam/system/lib/ppx_assert -I /home/opam/.opam/system/lib/ppx_bench -I /home/opam/.opam/system/lib/ppx_expect -I /home/opam/.opam/system/lib/ppx_inline_test -I /home/opam/.opam/system/lib/result -I /home/opam/.opam/system/lib/sexplib -I /home/opam/.opam/system/lib/typerep -I /home/opam/.opam/system/lib/variantslib -I /usr/lib/ocaml -I /usr/lib/ocaml/threads -no-alias-deps -I async -o async/faraday_async.cmi -c -intf async/faraday_async.mli)
# File "async/faraday_async.mli", line 9, characters 20-35:
# Error: Unbound module Deferred
```

File: packages/faraday-async/faraday-async.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [
+  ["jbuilder" "runtest" "-p" name]
+]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "faraday"
+  "async"
```
Comment (avsm) [inline packages/faraday-async/faraday-async.0.3.0/opam]: This should be fine if you add the following constraints:

```
"faraday" >= "0.3.0"
"async" >= "v0.9.0"
```

File: packages/faraday-lwt-unix/faraday-lwt-unix.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [
+  ["jbuilder" "runtest" "-p" name]
+]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "faraday"
+  "faraday-lwt"
```
Comment (avsm) [inline packages/faraday-lwt-unix/faraday-lwt-unix.0.3.0/opam]: These should also probably be:

```
"faraday" >= "0.3.0"
"faraday-lwt" >= "0.3.0"
```

File: packages/faraday-lwt/faraday-lwt.0.3.0/opam [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [
+  ["jbuilder" "runtest" "-p" name]
+]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "faraday"
```
Comment (avsm) [inline packages/faraday-lwt/faraday-lwt.0.3.0/opam]: This should be

```
"faraday" >= "0.3.0"
```

**Comment by samoht** [comment]
Thanks!

**Pull request merged**